src/Spout/Reader/XLSX/Helper/SharedStringsHelper.php

```diff
@@ -60,15 +60,28 @@ public function __construct($filePath, $tempFolder = null)
      */
     public function hasSharedStrings()
     {
-        $hasSharedStrings = false;
+        return $this->getSharedStringsPath() !== null;
+    }
+
+    /**
+     * Returns shared strings XML file path
+     *
+     * @return string|null
+     */
+    public function getSharedStringsPath()
+    {
+        $sharedStringsPath = null;
         $zip = new \ZipArchive();
 
         if ($zip->open($this->filePath) === true) {
-            $hasSharedStrings = ($zip->locateName(self::SHARED_STRINGS_XML_FILE_PATH) !== false);
+            $index = $zip->locateName(self::SHARED_STRINGS_XML_FILE_PATH, \ZipArchive::FL_NOCASE);
```

Comment:
This seems a more robust approach (in case, someone decides to change the name of the file alltogether)